public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -359,7 +361,10 @@ export class LokiDatasource
       return this.runLiveQueryThroughBackend(fixedRequest);
     }
 
-    if (config.featureToggles.lokiQuerySplitting && requestSupportsSplitting(fixedRequest.targets)) {
+    const lokiShardSplitting = true;
+    if (lokiShardSplitting && requestSupportsSharding(fixedRequest.targets)) {
```

Comment:
Can we limit this to Explore for now as well?

```suggestion
    if (lokiShardSplitting && requestSupportsSharding(fixedRequest.targets) && fixedRequest.app === CoreApp.Explore) {
```

Reply:
Definitely. Done.